docs/topics/testing/tools.txt

```diff
@@ -1599,6 +1599,36 @@ your test suite.
             Person.objects.create(name="Aaron")
             Person.objects.create(name="Daniel")
 
+.. _topics-tagging-tests:
+
+Tagging tests
+-------------
+
+You can tag your tests to group them into module-independent pieces- for example fast and slow tests::
```

Comment:
Not counting columns, but should this line be wrapped?

Also, I'm not sure the phrase "into module-independent pieces" gains very much; could maybe strike it.

And I think the separator for the "for example" clause should be ` -- `